src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import styles from './Toast.st.css';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { CloseIconButton } from './CloseIconButton/CloseIconButton';
+
+export enum TOAST_SKIN {
+  status = 'status',
+  success = 'success',
+  error = 'error',
+}
+
+export interface ToastProps {
+  /** Define the styles for toast */
+  skin: TOAST_SKIN;
+  shouldShowCloseButton?: boolean;
+  /** Callback function, will be called when click on close button */
+  onClose?(): void;
+}
+
+export interface DefaultProps {
+  shouldShowCloseButton: boolean;
+}
+
+interface State {}
+
+export class Toast extends React.Component<ToastProps, State> {
+  static displayName = 'Toast';
+
+  static defaultProps: DefaultProps = {
+    shouldShowCloseButton: false,
+  };
+
+  handleOnCloseClick = () => {
+    const { onClose } = this.props;
+    onClose && onClose();
+  };
+
+  render() {
+    const { skin, shouldShowCloseButton, children, ...rest } = this.props;
+
+    return (
+      <TPAComponentsConsumer>
+        {({ mobile, rtl }) => (
+          <div
+            {...styles('root', { mobile, rtl, skin }, rest)}
+            data-skin={skin}
+          >
+            <span className={styles.gapBeforeMessage}></span>
```

Comment:
Can you remove the two `span` elements and apply padding to `.root` element?

Reply:
I told Nataliia to do this, so it's ok.
It's in order to easily support rtl and use `flex`'s native support for rtl

Reply:
so we should use `role="presentation"` for these `span` elements (which tells the screen reader to ignore those elements)

Reply:
@giladsegal Are you mean gapBefore and gapAfter? We can not remove it, because we have not symmetric left-right padding. So for saving correct rtl, we (I mean @jonathanadler and me) decided to add gap-element.